packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.routing.js

```diff
@@ -9,6 +9,8 @@ export default /* @ngInject */ ($stateProvider) => {
         },
       },
       resolve: {
+        isConfiguration: /* @ngInject */ ($transition$) =>
+          $transition$.params().isConfiguration,
```

Comment:
```suggestion
        deviceId: /* @ngInject */ ($transition$) =>
          $transition$.params().deviceId,
```
Maybe you can use the deviceId as params ? If you have it's a configuration if not it's new add device